chat-plugins/wifi.js

```diff
@@ -589,10 +589,11 @@ let commands = {
 		this.privateModCommand(`(${user.name} started a GTS giveaway for ${targetUser.name})`);
 	},
 	left: function (target, room, user) {
-		if (room.id !== 'wifi' || !target) return false;
+		if (room.id !== 'wifi') return false;
+		if (!this.runBroadcast()) return this.errorReply("Permission denied");
```

Comment:
[8:30 PM] bumbadadabum: so you should do if (!this.runBroadcast()) return this.errorReply("Permission denied"); or something

Oops that one's on me actually sorry @Sipanale forgot that posted a message.